locales/en/translation.json

```diff
@@ -1,34 +1,961 @@
+  "Scan Addresses for Funds": "Scan Addresses for Funds",
+  "View all addresses": "View all addresses",
+  "Wallet Inputs": "Wallet Inputs",
+  "Total wallet inputs": "Total wallet inputs",
+  "Low amount inputs": "Low amount inputs",
+  "Approximate Bitcoin network fee to transfer wallet's balance (with normal priority)": "Approximate Bitcoin network fee to transfer wallet's balance (with normal priority)",
+  "Addresses with balance": "Addresses with balance",
+  "Unused addresses": "Unused addresses",
+  "All Addresses": "All Addresses",
+  " Wallet \"\" Addresses\nOnly Main Addresses are shown.\n\n\n'": " Wallet \"\" Addresses\nOnly Main Addresses are shown.\n\n\n'",
+  "Send Addresses by Email": "Send Addresses by Email",
+  "Export Wallet": "Export Wallet",
+  " (No Private Key)": " (No Private Key)",
+  "BitPay Wallet Backup: ": "BitPay Wallet Backup: ",
+  "Here is the encrypted backup of the wallet : \n\n \n\nTo import this backup, copy all text between {...}, including the symbols {}": "Here is the encrypted backup of the wallet : \n\n \n\nTo import this backup, copy all text between {...}, including the symbols {}",
+  "Export your asset by creating a password": "Export your asset by creating a password",
+  "Do not include private key": "Do not include private key",
+  "Send by Email": "Send by Email",
+  "Wallet Information": "Wallet Information",
+  "Name (at creation)": "Name (at creation)",
+  "WalletId": "WalletId",
+  "Linked Ethereum Wallet": "Linked Ethereum Wallet",
+  "Configuration (m-n)": "Configuration (m-n)",
+  "Address Type": "Address Type",
+  "Derivation Path": "Derivation Path",
+  "Read Only Wallet": "Read Only Wallet",
+  "No private key": "No private key",
+  "Copayers": "Copayers",
+  "(Me)": "(Me)",
+  "Extended Public Keys": "Extended Public Keys",
+  "Copayer ": "Copayer ",
+  "Balance By Address": "Balance By Address",
+  "Request a specific amount": "Request a specific amount",
+  "This will generate an invoice, which the person you send it to can pay using any wallet.": "This will generate an invoice, which the person you send it to can pay using any wallet.",
+  "Share Address": "Share Address",
+  "Share your wallet address to someone in your contacts so they can send you funds.": "Share your wallet address to someone in your contacts so they can send you funds.",
+  "Wallet Settings": "Wallet Settings",
+  "View all the ways to manage and configure your wallet.": "View all the ways to manage and configure your wallet.",
+  "Miner fee notice": "Miner fee notice",
+  "Because you are speeding up this transaction, the Bitcoin miner fee () will be deducted from the total.": "Because you are speeding up this transaction, the Bitcoin miner fee () will be deducted from the total.",
```

Comment:
`Because you are speeding up this transaction, the Bitcoin miner fee ({{speedupFee}} {{currencyAbbreviation}}) will be deducted from the total.`